doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,256 @@
+    { 'file', 'image', 'raster' },
+    { 'file', 'image' },
+    { 'file' }                                      # File
+]
+```
+
+Resulting in:
+
+```python
+policies = [
+    {},                    # Main document
+    {},                    # Generated Images
+    {},
+    {},
+    { 'managed': {...} },  # File
+]
+```
+
+#### Pros
+
+- Conveys only what traits are supported to the host, allowing
+  behaviour to be properly adapted to the manager's capabilities.
+
+#### Cons
+
+- Host implementation is significantly more complicated.
+- No way to hint at any additional traits that may be resolvable
+  outside the typing trait set.
+
+### Option 3 - Supported Traits Response
+
+The host should also include any additional traits that it may attempt
+to resolve in the future, e.g. if it supported the ability to customise
+t,he file format used for writing images, by resolving a custom trait:
+
+```python
+traitSets =[
+    { 'file', 'nodeGraph' },
+    { 'file', 'image', 'raster', 'colorManaged', 'fileFormatOptions' }
+]
+```
+
+The manager should respond as per Option 1, but in addition, imbue any
+of the queried traits that the manager is capable of resolving:
+
+```python
+if FileTrait.kId in traitSet:
+    ManagerTrait.imbueTo(policy)
+    FileTrait.imbueTo(policy)
+```
+
+This would then look like the following:
+
+```python
+policies = [
+    { 'managed': {...}, 'file': {} },  # Main document
+    { 'managed': {...}, 'file': {} }   # Generated Images
+]
+```
+
+This indicates that the manager would like to handle interactions for
+entities with these traits sets, but it can only resolve the `file`
+trait.
+
+> Note:
+> We keep the concept additionally imbuing the `managed` trait
+> (though it may seem superfluous, as an empty trait set is equivalent to
+> un-managed) as in practical use cases it may well have other properties
+> that cover topics such as exclusivity/etc.
+> There are also additional traits that may be imbued by the manager to
+> determine how it handles various aspects of the publishing process,
+> e.g. thumbnails.
+
+#### Pros
+
+- Explicit communication of exact capabilities, so hosts and managers
+  can suitably adapt behaviour.
+- Facilitates functional workflows where the result of `managementPolicy`
+  can be passed as the requested trait set for `resolve`.
+
+#### Cons
+
+- Implementation in the manager and host may be slightly more involved
+  than other options in some scenarios.
+
+## Outcome
+
+The `managementPolicy` mechanism should always be queried with the full
+trait set for any given entity specification, and any additional traits
+a host may attempt to resolve/publish for that type (if known).
+
+The response populated by a manager should include all
+resolvable/persisted traits from that set, along with any additional
+traits that describe the manager's behaviour.
+
+## Rationale
+
+This offers the best of both worlds. It provides the highest quality
```

Comment:
[Consider][Minor] Feel silly calling this out, but this might be an idiom that dosen't translate well.

Reply:
Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/792/commits/64249482a55a2163f83be86e5a6b30c15c129039).